packages/integrations/react/src/index.ts

```diff
@@ -17,7 +18,7 @@ function getRenderer() {
 						{},
 						{
 							runtime: 'automatic',
-							importSource: '@astrojs/react',
+							importSource: ReactVersion.startsWith('18.') ? 'react' : '@astrojs/react',
```

Comment:
In general this all LGTM, but is this line intentional? Why does this toggle between importing the integration and importing React itself here?

Reply:
Yep, mind explaining what this is doing @FredKSchott ?